### PR TITLE
feat(techdocs): use entity title as label in header

### PR DESCRIPTION
**Pull request by kuangp**
Signed-off-by: Phil Kuang <pkuang@factset.com>

## Hey, I just made a Pull Request!

| Before | After |
| ------- | ---- |
| Screen Shot 2022-06-01 at 10 50 17 AM | Screen Shot 2022-06-01 at 10 49 57 AM |

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/polite-ligers-hang.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-techdocs-addons-test-utils': patch
```
Comment (Rugvip) [inline .changeset/polite-ligers-hang.md:3]: That's not really the change to this package though? 😅 Probably best to split this into two changesets

**Review by Rugvip — APPROVED** [review]
Thanks! 👍 

🟢 to unblock

**Comment by kuangp** [comment]
@iamEAP or @emmaindal mind taking a quick look when you get a chance 😁  🙏

**Review by iamEAP — APPROVED** [review]
Thanks @kuangp!

**Pull request merged**